.storybook/main.js

```diff
@@ -1,8 +1,4 @@
-import type { StorybookViteConfig } from "@storybook/builder-vite";
-import { mergeConfig, PluginOption } from "vite";
-import viteConfig from "../vite.config";
-
-const storybookConfig: StorybookViteConfig = {
+module.exports = {
```

Comment:
I'm curious about what the bug you encountered was. Did it have to do with storyshots?

Reply:
Not exactly, there's a TypeScript issue that pops up when we try to remove Storybook's React plugin. It never seems to have been properly resolved in v6.x.

```sh
.storybook/main.ts:57:17 - error TS2339: Property 'name' does not exist on type 'Plugin_2 | Promise<false | Plugin_2 | PluginOption[]>'.
  Property 'name' does not exist on type 'Promise<false | Plugin_2 | PluginOption[]>'.

57   return plugin.name === name;
```

Minna even mentioned it back in the day here 😆 https://github.com/evergreen-ci/parsley/pull/6